### DISCUSSION TITLE
CS:GO社区服务器无法穿透

### DISCUSSION THREAD

**Discussion by ms12667** [natfrp/wiki#99] [使用问题]
我按照官方教程，尝试穿透csgo社区服务器，但我遇到一些问题
![image](https://user-images.githubusercontent.com/108612932/177036143-f38face6-3a5c-476b-b901-e588b08b966a.png)
我认为此处sv_lan不应为 1 ，否则服务器只能被局域网访问
![I1 E~F 1U$O$P5XU2~I}F~3](https://user-images.githubusercontent.com/108612932/177036362-3f2826df-dcca-41a6-878c-448e173d9159.png)
此外，当我尝试通过公网连接服务器时，服务器在公网可见，却无法进入服务器（如果没有成功控制台会一直提示retrying）。点击连接服务器提示连接一下然后弹回大厅（已排除 Steam 游戏服务器帐户令牌问题）
![SXXTW0$NO7AZOF92 ZLKCLD](https://user-images.githubusercontent.com/108612932/177036350-ede908a2-c75e-4344-bc7b-c8467a6154ac.png)
经过多方专业人士询问确认，该问题可能为映射问题，随后我尝试更换多处节点，与教程完全一致，问题依然无法解决，port为12665，希望得到解答
![}X UF8RNTN6K8I( 1BKFDMM](https://user-images.githubusercontent.com/108612932/177036578-2cc0be42-8229-4639-82c2-e6100d37d8c7.png)

**Comment by fakeboboliu** [comment]
我们希望你不要穿透 CSGO 服务器，因此我们作为官方此处不提供支持。

如果你对这些来自社区的文档有看法，我们很欢迎 [Pull Request](https://github.com/natfrp/wiki/pulls)。
